src/content/community/research/index.md

```diff
@@ -5,393 +5,439 @@ sidebar: true
 lang: en
 ---
 
-# Active areas of Ethereum research
+# Active areas of Ethereum research {#active-areas-of-ethereum-research}
 
-One of the primary strengths of Ethereum is that an active research and engineering community are constantly improving it. Many enthusiastic, skilled people worldwide would like to apply themselves to outstanding issues in Ethereum, but it is not always easy to find out what those issues are. This page outlines key active research areas as a rough guide to Ethereum's cutting edge.
+One of the primary strengths of Ethereum is that it is constantly being improved by an active research and engineering community. There are many enthusiastic, skilled people worldwide that would like to apply themselves to outstanding issues in Ethereum, but it is not always easy to find out what those issues are. This page aims to outline key areas of active research as a rough guide to Ethereum's cutting edge.
```

Comment:
Not sure why these copy changes are in here - perhaps the file is outdated? Could you please sync this up with latest in `dev` branch, then re-apply the header ids?